src/components/ClusterTemplateWizard/Steps/InstallationSettingsStep/DestinationNamespaceField.tsx

```diff
@@ -20,21 +20,25 @@ const getNamespaceOptions = (namespaces: string[]): SelectInputOption[] =>
 const DestinationNamespaceField = () => {
   const [namespaces, loaded, error] = useNamespaces();
   const useInstanceNamespaceFieldName = 'installation.useInstanceNamespace';
-  const namespaceFieldName = 'installation.spec.destinationNamespace';
+  const namespaceFieldName = 'installation.destinationNamespace';
```

Comment:
can you move these contstants outside of the function ? It makes it a bit more cleaner